phoenix-hive-base/pom.xml

```diff
@@ -583,4 +583,15 @@
       </plugins>
     </pluginManagement>
   </build>
+
+  <repositories>
+    <repository>
+      <id>saliyunmaven</id>
+      <name>aliyunmaven</name>
+      <url>https://maven.aliyun.com/repository/spring-plugin</url>
```

Comment:
Why do you prefer this repository than https://public.nexus.pentaho.org/repository/proxy-public-3rd-party-release ?

Reply:
The precommit test on your change takes way too much time. https://github.com/apache/phoenix-connectors/pull/53/checks 
I wanted to test if using this repo is faster. Locally it was better.